src/stores/OverlayStore/OverlayStore.ts

```diff
@@ -1157,18 +1180,53 @@ export class OverlayStore {
         return astString.toString();
     }
 
+    public channelMapInnerStyleString(frame?: FrameStore) {
+        let astString = new ASTSettingsString();
+        astString.addSection(this.global.styleString(frame));
+        astString.addSection(this.grid.styleString);
+        astString.addSection(this.border.styleString);
+        astString.addSection(this.ticks.styleString);
+        astString.addSection(this.axes.styleString);
+        astString.addSection(this.numbers.styleString);
+        astString.add("DrawTitle", false);
+        astString.add("TextLab", false);
+        astString.add("LabelUp", 0);
+        astString.add("TitleGap", 0);
+        astString.add("NumLabGap", this.defaultGap / this.minSize(frame));
+        astString.add("TextLabGap", 0);
+        astString.add("TextGapType", "plot");
+        return astString.toString();
+    }
+
+    public channelMapOuterStyleString(frame?: FrameStore) {
+        let astString = new ASTSettingsString();
+        astString.addSection(this.global.styleString(frame));
+        astString.addSection(this.title.styleString);
+        astString.addSection(this.labels.styleString);
+        astString.add("Grid", false);
+        astString.add("Border", false);
+        astString.add("MajTickLen(1)", 0);
+        astString.add("MinTickLen(1)", 0);
+        astString.add("MajTickLen(2)", 0);
+        astString.add("MinTickLen(2)", 0);
+        astString.add("DrawAxes", false);
+        astString.add("NumLab", false);
+        astString.add("LabelUp", 0);
+        astString.add("TitleGap", this.titleGap / Math.min(this.fullViewWidth - this.paddingLeft - this.paddingRight, this.fullViewHeight - this.paddingTop - this.paddingBottom));
+        astString.add("NumLabGap", 0);
+        astString.add("TextLabGap", this.cumulativeLabelGap / Math.min(this.fullViewWidth - this.paddingLeft - this.paddingRight, this.fullViewHeight - this.paddingTop - this.paddingBottom));
+        astString.add("TextGapType", "plot");
+        return astString.toString();
+    }
+
     @action minSize(frame?: FrameStore) {
-        return Math.min(frame.renderWidth || this.renderWidth, frame.renderHeight || this.renderHeight);
+        return Math.min(this.renderWidth || frame.renderWidth, this.renderHeight || frame.renderHeight);
     }
 
     @computed get showNumbers() {
         return this.numbers.show && this.global.labelType === LabelType.Exterior;
     }
 
-    @computed get defaultGap() {
```

Comment:
I prefer we set `OverlayStore.defaultGap` back to 5 instead of 2 to preserve the current paddings of the image view.